docs/Learning-Environment-Create-New.md

```diff
@@ -499,11 +499,12 @@ Now you can train the Agent. To get ready for training, you must first to change
 the `Brain` of the agent to be the Learning Brain `RollerBallBrain`.
 Then, select the Academy GameObject and check the `Control` checkbox for 
 the RollerBallBrain item in the **Broadcast Hub** list. From there, the process is
-the same as described in [Training ML-Agents](Training-ML-Agents.md).
+the same as described in [Training ML-Agents](Training-ML-Agents.md). Note that the config
+files and models will be created in the original ml-agents project folder.
```

Comment:
can you create a reference to the folder starting with root (i.e. \ml-agents\...)